plugins/kubernetes-backend/src/service/KubernetesProxy.test.ts

```diff
@@ -1014,18 +1016,18 @@ describe('KubernetesProxy', () => {
   describe('Backstage running on k8s', () => {
     const initialHost = process.env.KUBERNETES_SERVICE_HOST;
     const initialPort = process.env.KUBERNETES_SERVICE_PORT;
-    const initialCaPath = Config.SERVICEACCOUNT_CA_PATH;
+    const initialCaPath = SERVICEACCOUNT_CA_PATH;
 
     afterEach(() => {
       process.env.KUBERNETES_SERVICE_HOST = initialHost;
       process.env.KUBERNETES_SERVICE_PORT = initialPort;
-      Config.SERVICEACCOUNT_CA_PATH = initialCaPath;
+      SERVICEACCOUNT_CA_PATH = initialCaPath;
     });
 
     it('makes in-cluster requests when cluster details has no token', async () => {
       process.env.KUBERNETES_SERVICE_HOST = '10.10.10.10';
       process.env.KUBERNETES_SERVICE_PORT = '443';
-      Config.SERVICEACCOUNT_CA_PATH = mockCertDir.resolve('ca.crt');
```

Comment:
Hey this is the problem with the test, we are no longer injecting the mocked directory into `KubernetesProxy.ts` via overriding the `SERVICEACCOUNT_CA_PATH` of `Config`, which we should have probably never done.

Reply:
Config is no longer available from the kubernetes/client-node library. Any suggestions?

Reply:
I guess some refactoring so we can inject the mocked value in? We could import the value from a constant file and mock that import

Reply:
I `think` that I have resolved this, ptal.